src/app/_translate/abbreviate-number.pipe.ts

```diff
@@ -0,0 +1,24 @@
+/** AbbreviateNumberPipe
```

Comment:
What is this class used for?

Reply:
It's a pipe for abbreviating numbers.  Comment added.